src/main/java/org/broadinstitute/dsm/util/KitUtil.java

```diff
@@ -271,7 +281,9 @@ private static void buyShipmentForKit(@NonNull EasyPostUtil easyPostUtil, @NonNu
         catch (Exception e) {
             errorMessage += "Return: " + e.getMessage();
         }
-        KitRequestShipping.updateKit(dsmKitId, participantShipment, returnShipment, errorMessage, toAddress, false);
+        DDPInstanceDao ddpInstanceDao = new DDPInstanceDao();
+        DDPInstanceDto ddpInstanceDto = ddpInstanceDao.getDDPInstanceByInstanceId(kitRequestSettings.getDdpInstanceId()).orElseThrow();
```

Comment:
we already have a ddpInstanceDto object in the method that calls buyShipmentForKit, could we reuse that one